lib/rules/jsx-needs-i18n.js

```diff
@@ -9,43 +9,50 @@
 // Rule Definition
 // ------------------------------------------------------------------------------
 
-module.exports = function(context) {
+module.exports = {
+    meta: {
+        docs: {},
+        schema: [{
+            type: 'string'
+        }]
+    },
+
+    create: function(context) {
+
+        function reportLiteralNode(node) {
+            context.report(node, 'Missing translation function around string literal');
+        }
 
-    function reportLiteralNode(node) {
-        context.report(node, 'Missing translation function around string literal');
-    }
+        // default: must have at least one letter; not symbols, numbers
+        const testRegExp = context.options[0] ? new RegExp(context.options[0]) : /[A-Za-z]/;
```

Comment:
In my project, literals are written in Japanese, so the default regexp /[A-Za-z]/ does not match like following code:

```html
<button className="button-default is-action">保存</button>
```